Department.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Main;
+
+use ArrayObject;
+
+class Department
+{
+    private string $name;
+    private array $workers;
+    public function __construct($name, $list)
+    {
+        $this->name=$name;
+        $this->workers=new ArrayObject($list);
+    }
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    public function getWorkers()
```

Comment:
- added return type hinting for getWorkers method
```suggestion
    public function getWorkers(): ArrayObject
```